MorphicServer/AuthUsernamePasswordResetEndpoint.cs

```diff
@@ -158,33 +170,49 @@ public async Task Post()
             {
                 throw new HttpError(HttpStatusCode.BadRequest, BadPasswordRequestResponse.MissingRequired);
             }
+
+            if (!User.IsValidEmail(request.Email))
```

Comment:
Do we really need this check.  It doesn't do any harm, but if it's a malformed email it won't match a user, right?

Reply:
Correct. I figured better error messages help, but I can remove it.

Reply:
Ah... if the email isn't a good one, then we can't later send an email (see "BackgroundJob.Enqueue<EmailNotVerifiedPasswordResetEmail>..."), either, so we might as well abort here (and get a better message). This follows best-practices, alerting a user that they might have entered an email we don't have a user for (that's better than a "user not found for this email" error message).